src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/widget/VKWidgetPageObject.java

```diff
@@ -11,13 +13,17 @@
 
   @FindBy(css = ".widget-vk iframe")
   private WebElement vkIframe;
+  @FindBy(css = ".widget-vk iframe")
+  private List<WebElement> vkIframeList;
   @FindBy(css = ".widget_body")
   private WebElement vkBody;
 
   private static final String TAG_NAME = "vk";
   private static final String ARTICLE_NAME = "VKWidget";
-  private static final String TAG =
-      "<vk group-id=\"12345\" />";
+  private static final String TAGS[] = {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move the array designator from the variable to the type. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1197)